Sources/eppo/ConfigurationRequester.swift

```diff
@@ -14,3 +18,15 @@ class ConfigurationRequester {
         return try UniversalFlagConfig.decodeFromJSON(from: String(data: urlData, encoding: .utf8)!);
     }
 }
+
+class JsonConfigurationRequester: ConfigurationRequesterProtocol {
+    private let configurationJson: String;
+
+    public init(configurationJson: String) {
+        self.configurationJson = configurationJson
+    }
+
+    public func fetchConfigurations() throws -> UniversalFlagConfig {
+        return try UniversalFlagConfig.decodeFromJSON(from: configurationJson);
+    }
```

Comment:
minor: this will trigger JSON-decoding on every call to `fetchConfigurations()` instead of only once